notification2-examples/hello-world-microservice/src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 application.name=hello-notification
 server.port=8085
 
-example.source.id=29100
+example.source.id=883202
 example.websocket.url=ws://cumulocity.default.svc.cluster.local
```

Comment:
I would also try to remove this one and obtain the url from one of the Java SDK beans maybe from the `Platform`?

Reply:
Exclusively from tenant properties you mean?

Reply:
Let's leave it as it is as this is not really in scope of this PR